specs/deneb/polynomial-commitments.md

```diff
@@ -170,6 +171,12 @@ def bytes_to_bls_field(b: Bytes32) -> BLSFieldElement:
     return BLSFieldElement(field_element)
 ```
 
+#### `bls_field_to_bytes`
+
+```python
+def bls_field_to_bytes(x: BLSFieldElement) -> Bytes32:
+    return int.to_bytes(x % BLS_MODULUS, 32, KZG_ENDIANNESS)
```

Comment:
Doesn't `BLSFieldElement` type imply that `x < BLS_MODULUS`? 
What is the meaning of `% BLS_MODULUS` operation then?

Reply:
it should, but the way the typing is set up a `BLSFieldElement` is any possible `uint256` and so the reduction is necessary